include/net/nrf_cloud_rest.h

```diff
@@ -68,7 +68,7 @@ struct nrf_cloud_rest_context {
 	 * Minimum timeout value specified by NRF_CLOUD_REST_TIMEOUT_MINIMUM.
 	 * For no timeout, set to NRF_CLOUD_REST_TIMEOUT_NONE.
 	 * @note This parameter is currently not used; set
-	 * CONFIG_NRF_CLOUD_REST_RECV_TIMEOUT instead.
+	 * CONFIG_REST_CLIENT_SCKT_RECV_TIMEOUT instead.
 	 */
 	int32_t timeout_ms;
```

Comment:
Note: not used?  is it due to that http_client is having issues with the timeout? At least I have seen that. That was also marked as known issues for REST client / Location API, also a fresh Jira bug was created (NCSDK-13210).

Reply:
correct.